app/controllers/profile/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Profile::DashboardController < ApplicationController
+  before_action :require_regular
+
+  def index
+  end
+
+  private
+
+  def require_regular
```

Comment:
Why does this method need to be private?

Reply:
I made it private to follow the convention here: https://backend.turing.io/module2/lessons/authorization (Cmd+F "private")